src/libraries/System.Text.Json/tests/Resources/Issue42070_0004.json

```diff
@@ -0,0 +1 @@
kinson","address2":"transmitter","company":"New Hampshire","latitude":0.9235264,"longitude":0.35687974,"name":"Concrete","country_code":"PL","province_code":"bus"},{"first_name":"Cydney","address1":"Plastic","phone":"Ergonomic Cotton Ball","city":"Runtetown","zip":"Future","province":"Agent","country":"Palau","last_name":"Ritchie","address2":"leading edge","company":"Money Market Account","latitude":0.5705042,"longitude":0.85383874,"name":"Unbranded Granite Cheese","country_code":"HU","province_code":"Comoro Franc"},{"first_name":"Alexis","address1":"Schuster Lake","phone":"Denar","city":"Bradtkeland","zip":"e-tailers","province":"Maryland","country":"Congo","last_name":"Green","address2":"Kyle Harbors","company":"Intranet","latitude":0.2534823,"longitude":0.42616767,"name":"invoice","country_code":"LA","province_code":"Savings Account"}]},{"id":-2450321883376302498,"email":"Alexis_Thiel66@hotmail.com","accepts_marketing":true,"created_at":"2020-09-15T20:39:51.9607651+01:00","updated_at":"2020-09-16T11:19:23.5700914+01:00","first_name":"Erik","last_name":"Tromp","orders_count":-263155800,"state":"Hawaii","total_spent":"Station","last_order_id":-6693628180844590522,"note":"Guinea-Bissau","verified_email":false,"multipass_identifier":"Savings Account","tax_exempt":false,"tags":"application","last_order_name":"Berkshire","default_address":{"first_name":"Lane","address1":"Joey Vista","phone":"invoice","city":"West Ryleighburgh","zip":"back-end","province":null,"country":"Turkey","last_name":"Rippin","address2":null,"company":"Coves","latitude":0.22509784,"longitude":0.16279897,"name":"Berkshire","country_code":"NA","province_code":"analyzer"},"addresses":[{"first_name":"Drake","address1":"Lights","phone":"cyan","city":"Kuvalisfort","zip":"Practical Concrete Sausages","province":"Engineer","country":"Myanmar","last_name":"West","address2":"robust","company":"Chief","latitude":0.33437592,"longitude":0.18548732,"name":"Home Loan Account","country_code":"ME","province_code":"Intelligent Granite Table"},{"first_name":"Quentin","address1":"integrated","phone":"Cambridgeshire","city":"North Lilian","zip":"compress","province":"TCP","country":"Gibraltar","last_name":"Gleason","address2":"Accounts","company":"Money Market Account","latitude":0.9771962,"longitude":0.59250027,"name":"tertiary","country_code":"DK","province_code":"Buckinghamshire"},{"first_name":"Jammie","address1":"matrix","phone":"deploy","city":"Grahamborough","zip":"RAM","province":"Utah","country":"Iceland","last_name":"Rau","address2":"convergence","company":"exploit","latitude":0.6584244,"longitude":0.3303229,"name":"capacity","country_code":"IL","province_code":"Open-architected"},{"first_name":"Lane","address1":"Joey Vista","phone":"invoice","city":"West Ryleighburgh","zip":"back-end","province":null,"country":"Turkey","last_name":"Rippin","address2":null,"company":"Coves","latitude":0.22509784,"longitude":0.16279897,"name":"Berkshire","country_code":"NA","province_code":"analyzer"}]}]}
```

Comment:
We store other test payloads in this [`Strings.resx` file](https://raw.githubusercontent.com/dotnet/runtime/a7b9c36d19bfddef73339e7a59f03e27bdd7b1f6/src/libraries/System.Text.Json/tests/Resources/Strings.resx). Based on some other entries, the data name should be `SomeDescriptivePayloadSize` e.g. `BigPayloadWithNulls101KB`.

The usage would be `string json = SR.BigPayloadWithNulls101KB;`